### PR TITLE
Handle new 'main' branch in dotnet/aspnetcore

### PR DESCRIPTION
**Pull request by dougbu**
- part of dotnet/aspnetcore#29475

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -634,8 +634,7 @@
       "triggerPaths": [
         "https://github.com/aspnet/AspLabs/blob/master/**/*",
         "https://github.com/aspnet/AspLabs/blob/release/**/*",
-        "https://github.com/dotnet/aspnetcore/blob/blazor-wasm/**/*",
-        "https://github.com/dotnet/aspnetcore/blob/master/**/*",
+        "https://github.com/dotnet/aspnetcore/blob/main/**/*",
```
Comment (dougbu) [inline Maestro/subscriptions.json:637]: No longer have a blazor-wasm branch and will not have a main branch a bit after 5PM

File: Maestro/subscriptions.json [hunk 2]
```diff
@@ -1047,7 +1042,10 @@
     },
     {
       "triggerPaths": [
-
+        "https://github.com/dotnet/aspnetcore/blob/release/2.1//**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/3.1//**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/5.0//**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/6.0*//**/*",
```
Comment (dougbu) [inline Maestro/subscriptions.json:1048]: Merge all of these to main (along w/ dotnet/efcore) instead of master

**Review by dougbu — COMMENTED** [review]
/cc @dagood @mmitche @wtgodbe

/fyi @dotnet/aspnet-build

**Comment by dougbu** [comment]
I won't merge this until the right part of the transition process tonight&hellip;

**Comment by dougbu** [comment]
Oops, may need someone around to do the actual merge since I lack permissions in this repo. Will anyone be available this evening❔

**Comment by wtgodbe** [comment]
@mmitche do you have permissions to give Doug permissions? I do not

**Comment by dagood** [comment]
> @mmitche do you have permissions to give Doug permissions? I do not

@dotnet/dotnet-versions-admin 
(For future reference, owners/permissions show up at https://repos.opensource.microsoft.com/orgs/dotnet/repos/versions.)

I'm adding you as a writer, @dougbu. This looks fine and you've done a bunch of these PRs before. 🙂

We should probably work out with dnceng how to handle dotnet/versions permissions to streamline the m2m work. But we should probably avoid adding *too* much risk of someone accidentally merging a broken JSON file and putting Maestro offline.

**Comment by dougbu** [comment]
Thanks @dagood❕

**Comment by lukas-lansky** [comment]
> We should probably work out with @dotnet/dnceng how to handle dotnet/versions permissions to streamline the m2m work. But we should probably avoid adding too much risk of someone accidentally merging a broken JSON file and putting Maestro offline.

@premun @jakubstilec, what do you think?

**Comment by jakubstilec** [comment]
I think it would be better if somebody from @dotnet/dnceng always checks and is around in a case of issues. We have our FR and covering big part of day.

**Comment by premun** [comment]
@jakubstilec I am not 100% sure all of dnceng have the permissions though. I know Tomas does, me and you, probably not (I don't even see the settings). We should add dnceng for the time being maybe? @tkapin?

**Comment by tkapin** [comment]
To close on the above, this has been logged by @premun as #665.

**Pull request merged**